frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/expectedkeys/IExpectsKeyGitMacheteRepository.java

```diff
@@ -13,7 +13,7 @@
 public interface IExpectsKeyGitMacheteRepository extends IWithLogger {
   default @Nullable IGitMacheteRepositorySnapshot getGitMacheteRepositorySnapshot(AnActionEvent anActionEvent) {
     IGitMacheteRepositorySnapshot gitMacheteRepositorySnapshot = anActionEvent != null
-        ? anActionEvent.getData(DataKeys.KEY_GIT_MACHETE_REPOSITORY_SNAPSHOT)
```

Comment:
Pls extract such refactors to a separate PR, this PR is gonna be quite sizable probably 😅

Reply:
good idea

Reply:
thanks